openapi.yml

```diff
@@ -132,12 +132,7 @@ paths:
         content:
           application/json:
             schema:
-              type: object
-              properties:
-                work_email:
-                  $ref: "#/components/schemas/EmailAddress"
-              required:
-                - work_email
+              $ref: "#/components/schemas/WorkEmailObj"
```

Comment:
If the `Obj` in `WorkEmailObj` stands for `Object` like in Object-Oriented-Programming
it's redundant.

As it's a container for user profile data, it may evolve in time with future properties.
If  the semantic of this request is the one requesting an *action* on a resource and not a full replacement,
the schema could be something like `UpdateUserProfile`.